.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ version: 2.1
 executors:
   golang:
     docker:
-      - image: docker.mirror.hashicorp.services/cimg/go:1.17
+      - image: cimg/go:1.17
```

Comment:
@andrewsomething I don't know that circleci is used within the org but I updated just in case. Feel free to delete.